lib/ash_authentication/strategies/password/request_password_reset.ex

```diff
@@ -25,6 +25,7 @@ defmodule AshAuthentication.Strategy.Password.RequestPasswordReset do
       identity = Ash.ActionInput.get_argument(action_input, identity_field)
       select_for_senders = Info.authentication_select_for_senders!(action_input.resource)
       {sender, send_opts} = strategy.resettable.sender
+      send_opts = Keyword.merge(send_opts, Map.to_list(action_input.context))
```

Comment:
`context` isn't guaranteed to have atom keys. I believe we should do this:
`send_opts = Keyword.put(send_opts, :context, action_input.context)`